editor-components/header/header5/header5.tsx

```diff
@@ -50,20 +91,44 @@ class Header5 extends BaseHeader {
   }
 
   render() {
+
+    const heading = this.castToObject<Heading>("heading");
+
+    
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("heading-page")}>
-            <h1 className={this.decorateCSS("heading-section-name")}>
-              {this.getPropValue("heading")[0].value}
-            </h1>
-            <h2 className={this.decorateCSS("heading-title")}>{this.getPropValue("heading")[1].value}</h2>
-            <h3 className={this.decorateCSS("heading-subtitle")}>{this.getPropValue("heading")[2].value}</h3>
-            <ComposerLink path={this.getPropValue("heading")[4].value}>
-              <span className={this.decorateCSS("button")}>
-                {this.getPropValue("heading")[3].value}
-              </span>
-            </ComposerLink>
+          <div
+            className={this.decorateCSS("background-layer")}
+            style={{
+              backgroundImage: `url(${heading.backgroundImage})`,
+            }}
+          >
+            <div className={heading.overlay && this.decorateCSS("overlay")}>
+              <div className={this.decorateCSS("heading-page")}>
+                <h1 className={this.decorateCSS("heading-section-name")}>{heading.titleColored}</h1>
+                <h2 className={this.decorateCSS("heading-title")}>{heading.title}</h2>
+                <h3 className={this.decorateCSS("heading-subtitle")}>{heading.description}</h3>
```

Comment:
if prop is empty, don't render relevant elements
[check here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)